packages/office-ui-fabric-react/src/common/IAccessiblePopupProps.ts

```diff
@@ -1,34 +1,38 @@
+/**
+ * {@docCategory Modal}
```

Comment:
Is `Modal` correct here? Curious since this is in a `common` directory.

Reply:
The options I thought of were either Modal, Dialog, or a separate docCategory which would land under References.

Reply:
I decided to change to a separate category which will go under References